example/App.js

```diff
@@ -159,6 +159,7 @@ export default class App extends Component {
       deviceJSON.userAgent = await DeviceInfo.getUserAgent();
       deviceJSON.instanceId = await DeviceInfo.getInstanceId();
       deviceJSON.installReferrer = await DeviceInfo.getInstallReferrer();
+      deviceJSON.installerPackageName = await DeviceInfo.getInstallerPackageName();
```

Comment:
I just realized (as I was thinking through where I'll merge this then test it) that now that you added the sync variant (thank you!) you need to add a the same above in the sync section, can you do that :pray:  then I'll merge, do a quick local build+check and release? It's the only thing I can see now, everything else looks in great shape

Reply:
not a problem, have added that example.